### DISCUSSION TITLE
How to split an initrd into multiple files ?

### DISCUSSION THREAD

**Discussion by leElvyn** [ipxe/ipxe#800] [Q&A]
Hi, 
I know this is a very specific issue, but i am trying to host a remote image that contains a kernel and an initial ram disk. I have small problem : it is a lot easier for me to host files under 400mb. It is easy to split files using the `split` commands, however, how can i recreate it in ipxe once the files are downloaded ?

**Comment by NiKiZe** [comment]
Maybe #705 can help answer parts of your question.
Also see https://ipxe.org/cmd/imgfetch

the Linux kernel will parse and combine all cpio archives that it knows about.
